conda-recipes/llvmlite/conda_build_config.yaml

```diff
@@ -1,14 +1,14 @@
 # Numba/llvmlite stack needs an older compiler for backwards compatability.
-c_compiler_version:         # [linux and osx]
+c_compiler_version:
```

Comment:
same here, use 'linux or (osx and x86_64)' ... the old line seems buggy anyway.  It should have been 'linux or osx'